addons/mail/static/src/new/web/messaging_menu/messaging_menu.xml

```diff
@@ -70,6 +70,10 @@
                     hasMarkAsReadButton="preview.hasMarkAsReadButton"
                     isLast="preview_last and failures.length === 0"
                     onClick="(isMarkAsRead) => this.onClickPreview(isMarkAsRead, preview)"
+                    onSwipeRight="() => this.onClickPreview(true, preview)"
```

Comment:
I believe click on preview opens it, I don't think swipe should open it, just mark as is read?

Reply:
Done (small-code cleaning to use `markAsRead(preview)`)